pkg/kubelet/network/cni/cni.go

```diff
@@ -189,6 +183,14 @@ func (plugin *cniNetworkPlugin) Name() string {
 	return CNIPluginName
 }
 
+func (plugin *cniNetworkPlugin) Status() error {
```

Comment:
How frequently is this called? Can we just add checkInitialized and not move this? What's the point?